tests/fsharp/typecheck/sigs/neg99.fs

```diff
@@ -0,0 +1,26 @@
+namespace CrashFSC
+
+open System
+
+// debatable code, but it was the minimalist example I could come up 
+// with after analysis in the original project
+module OhOh =
```

Comment:
@dsyme, may I suggest that we, for tests that originate from a bug report, add a comment back to that bug report? It may help to quickly find the cause whenever a committer gets into the same crash again if she/he modifies something wrongly. I do this in my own projects, and while it is not impossible to trace back by other means, it's just a tad more convenient that way.

And shouldn't we add the other testcases here as well? Or are they truly only duplicates?